### DISCUSSION TITLE
Quasar V2 - QInput with icon

### DISCUSSION THREAD

**Discussion by AlexAndrascu** [quasarframework/quasar#14103] [General - Components / Directives / etc] [answered]
I'm trying to append / prepend an icon to a q-input field like in the examples from [quasar-play](https://github.com/quasarframework/quasar-play/blob/master/src/pages/showcase/forms/input.vue) but it doesn't appear to be working. 

What am I doing wrong? 

[Codepen here](https://codepen.io/alexandrascu/pen/NWYazdg)

**Comment by AlexAndrascu** [comment]
It works if I use v-slots.

**Comment by yusufkandemir** [comment, marked as answer]
Those examples are 4 years old and use Quasar v0.17, while we are at Quasar 2.7.6. You can see the documentation for a lot of examples. For example, you can check this out: https://quasar.dev/vue-components/input#example--coloring
![image](https://user-images.githubusercontent.com/6266078/182160693-c763e598-a40f-4786-b126-b728e66a080c.png)
So, you should use slots like `append` and `prepend` to display icons.
